modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase

```diff
@@ -1013,6 +1013,143 @@ definition {
+			CommerceNavigator.gotoPortlet(
+				category = "Product Management",
+				portlet = "Options");
+
+			Click(
+				key_value = "New Option",
+				locator1 = "CommerceEntry#BODY_VERTICAL_ELLIPSIS");
+
+			CommerceEntry.clickNoError(menuItem = "Delete");
+		}
+
+		task ("When the admin goes back to the product") {
+			CommerceProducts.openProductsAdmin();
+
+			CommerceNavigator.searchEntry(entryName = "Simple T-Shirt");
+
+			CommerceNavigator.gotoEntry(entryName = "Simple T-Shirt");
+
+			CommerceEntry.gotoMenuTab(menuTab = "Options");
+		}
+
+		task ("Then the admin can create a new product option") {
+			CommerceProductOptions.createProductOption(key_optionName = "New Option1");
+		}
+	}
+
+	@description = "COMMERCE-11494. Can delete product option with generate All SKUs are created"
+	@priority = 4
+	test CanDeleteProductOptionWithGenerateAllSKUsCreated {
+		property custom.properties = "jsonws.web.service.paths.excludes=${line.separator}company.security.strangers.verify=false";
+		property portal.acceptance = "false";
+
+		task ("Given a Minium site is created") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+		}
+
+		task ("And an option is created") {
+			CommerceProducts.openProductsAdmin();
+
+			CommerceProducts.searchAndGoToProductEntry(entryProductName = "ABS Sensor");
+
+			CommerceEntry.gotoMenuTab(menuTab = "Options");
+
+			CommerceProductOptions.createProductOption(key_optionName = "Option1");
+
+			Refresh();
+
+			CommerceProductOptions.editProductOption(
+				key_fieldType = "Select from List",
+				key_option = "Option1",
+				productOptionFieldType = "true",
+				productOptionRequired = "true",
+				productOptionSkuContributor = "true");
+		}
+
+		task ("And 2 values are added to the option") {
+			for (var valueName : list "Value1,Value2") {
+				CommerceProductOptions.createProductOptionValue(key_name = ${valueName});
+
+				SelectFrame.selectFrameTop();
+
+				SelectFrame.selectFrameNoLoading(locator1 = "CommerceEntry#IFRAME_SIDE_PANEL");
+			}
+
+			SelectFrameTop();
+
+			Click(locator1 = "Icon#CLOSE");
+		}
+
+		task ("And from the SKUs tab, plus button is clicked, Generate all SKUs combination button is selected") {
+			CommerceEntry.gotoMenuTab(menuTab = "SKUs");
+
+			Click(locator1 = "Button#PLUS");
+
+			Click(
+				key_fieldName = "Generate All SKU Combinations",
+				locator1 = "FormViewBuilder#OBJECT_FIELD_BUTTON_DROPDOWN");
+		}
+
+		task ("When the option is deleted") {
+			CommerceEntry.gotoMenuTab(menuTab = "Options");
+
+			SelectFrameTop();
+
+			Click(
+				key_value = "Option1",
+				locator1 = "CommerceEntry#BODY_VERTICAL_ELLIPSIS");
+
+			CommerceEntry.clickNoError(menuItem = "Delete");
+		}
+
+		task ("Then when the admin goes back to the products admin page and the product is displayed properly") {
+			CommerceProducts.openProductsAdmin();
+
+			CommerceNavigator.searchEntry(entryName = "ABS Sensor");
```

Comment:
Add a couple more assertions here, for example: check a couple products more, the number of products

Reply:
@Brunno0liveira please add assertions here, do not use `CommerceNavigator.searchEntry()` to assert.

Also, by looking at the bug's description, it says:
> Expected Result:  Disable the active SKU & All the product pages shows correctly all the products

You also need to check that the skus are disabled.